### PR TITLE
Update yarn deps to @iov/core v0.5.1

### PR DESCRIPTION
**Pull request by ethanfrey**
This allows us to get full subscription support and proper txid on all responses.

### CODE REVIEW

**Comment by ethanfrey** [comment]
I updated deps and fixed the breaking change with IovWriter construction.
I think it is good, and will base on this to update txid

File: src/prom/connectionManager.mjs [hunk 1]
```diff
@@ -2,19 +2,18 @@ import iovCore from '@iov/core';
...
 async function promMakeWriter (profile, networkList, extraOpts) {
-  const { bnsConnector, IovWriter, withConnectors } = iovCore;
+  const { bnsConnector, IovWriter } = iovCore;
   const chainIds = Object.keys(networkList);
   chainIds.sort();
-  const networks = chainIds.map(async chainId => {
-    const bnsNetwork = await bnsConnector(urljoin('wss://bov.'+networkList[chainId].url));
-    return bnsNetwork;
-  });
+  const networks = chainIds.map(chainId =>
```
Comment (rudi-cilibrasi) [inline src/prom/connectionManager.mjs]: thought i saw this part in the other pr already. nbd

File: src/prom/connectionManager.mjs [hunk 2]
```diff
@@ -2,19 +2,18 @@ import iovCore from '@iov/core';
...
 
-  const resolvedNetworks = await Promise.all(networks);
   let writer;
   try {
-    const chains = await withConnectors(resolvedNetworks);
-    writer = new IovWriter(profile, chains);
+    writer = new IovWriter(profile);
+    for (const network of networks) {
+      await writer.addChain(network);
+    }
```
Comment (rudi-cilibrasi) [inline src/prom/connectionManager.mjs]: reviewed and approved with minor concern in other pr

**Comment by ethanfrey** [comment]
Other PR was rebased on top of this one.
I will fix this one up first, I think this is where the issues lie

**Comment by ethanfrey** [comment]
@rudi-cilibrasi I fixed up addChain to use `Promise.all`. I will rebase other pr on top. If you like this change, please merge. I did test manually and query and sendTx on friendnet-slow.

**Pull request merged**